ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -203,7 +203,1503 @@ public final class GlobalTokens {
         }
     }
 
+    // MARK: - SharedColors
+
+    public enum SharedColorSets: CaseIterable {
+        case darkRed
+        case burgundy
+        case cranberry
+        case red
+        case darkOrange
+        case bronze
+        case pumpkin
+        case orange
+        case peach
+        case marigold
+        case yellow
+        case gold
+        case brass
+        case brown
+        case darkBrown
+        case lime
+        case forest
+        case seafoam
+        case lightGreen
+        case green
+        case darkGreen
+        case lightTeal
+        case teal
+        case darkTeal
+        case cyan
+        case steel
+        case lightBlue
+        case blue
+        case royalBlue
+        case darkBlue
+        case cornflower
+        case navy
+        case lavender
+        case purple
+        case darkPurple
+        case orchid
+        case grape
+        case berry
+        case lilac
+        case pink
+        case hotPink
+        case magenta
+        case plum
+        case beige
+        case mink
+        case silver
+        case platinum
+        case anchor
+        case charcoal
+    }
+
+    public enum SharedColorsTokens: CaseIterable {
+        case shade50
+        case shade40
+        case shade30
+        case shade20
+        case shade10
+        case primary
+        case tint10
+        case tint20
+        case tint30
+        case tint40
+        case tint50
+        case tint60
+    }
+
+    lazy public var sharedColors: TokenSet<SharedColorSets, TokenSet<SharedColorsTokens, ColorValue>> = .init { sharedColor in
+        switch sharedColor {
+        case .anchor:
+            return TokenSet<SharedColorsTokens, ColorValue>.init { token in
+                switch token {
+                case .primary:
+                    return ColorValue(0x394146)
```

Comment:
Same color across light and dark mode?

Reply:
Yep! These are constant values, and are unaware of the current appearance mode. It's expected that `AliasTokens` will use these as building blocks when filling out a `DynamicColor` instance with light and dark colors.